.travis.yml

```diff
@@ -11,7 +11,13 @@ install:
 
 services:
   - mongodb
-# make mongodb a replica set
+ddons:
```

Comment:
Copied the config from this issue: https://github.com/travis-ci/travis-ci/issues/3694 - but yeah, I removed a letter :D